tests/test-umockdev.c

```diff
@@ -1980,20 +2023,13 @@ t_testbed_replay_evemu_events(UMockdevTestbedFixture * fixture, UNUSED_DATA)
           "E: DEVNAME=/dev/input/event1\nE: SUBSYSTEM=input\n", &error);
   g_assert_no_error(error);
 
-  /* write evemu events file */
-  fd = g_file_open_tmp("test_evemu.XXXXXX", &tmppath, &error);
-  g_assert_no_error(error);
-  g_assert_cmpint(write(fd, test_data, strlen(test_data)), ==, strlen(test_data));
-  close(fd);
-
   /* load it */
-  success = umockdev_testbed_load_evemu_events(fixture->testbed, "/dev/input/event1", tmppath, &error);
+  success = umockdev_testbed_load_evemu_events_from_string(fixture->testbed, "/dev/input/event1", test_data, &error);
```

Comment:
Should I be able to call `umockdev_testbed_load_evemu_events_from_string()`more than once?
I'm getting `ERROR:../src/umockdev.vala:1086:umockdev_testbed_load_script_from_stream: assertion failed: (!this.dev_script_runner.contains (owned_dev))`

I think this error was the motivation for my hack.

Reply:
Yes, you should be able to do that. I just pushed an update which asserts that this works. However, in between two load calls you have to call `umockdev_testbed_wait_script()` to ensure that the previous script is finished. That will flush the script from the `this.dev_script_runner` dictionary.

Reply:
> update which asserts that this works

I.e. I added unit tests for that, I didn't change the code. If that doesn't work for you, can you please point me to example code that fails? Thanks!